hal/armv7a/zynq7000/_init.S

```diff
@@ -159,6 +171,21 @@ set_loop:
 	bx r8
 .size _start, .-_start
 
+.globl other_core_start
```

Comment:
Use module_camelCase style, in this case:
`hal_otherCoreStart`
or perhaps
`hal_coreStart`